testbot.js

```diff
@@ -736,6 +722,36 @@ function handleEditCommand(msg, args) {
     });
 }
 
+function validatePriceModifier(priceModifier, finalPrice = null) {
+    /*

```

Comment:
Docustrings so it is easy to understand what the function does, what arguments it takes and what it returns